community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/CypherCompiler.scala

```diff
@@ -62,6 +62,8 @@ case class CypherCompilerConfiguration(queryCacheSize: Int,
                                        statsDivergenceThreshold: Double,
                                        queryPlanTTL: Long,
                                        useErrorsOverWarnings: Boolean,
+                                       idpMaxTableSize: Int,
+                                       idpIterationDuration: Long,
```

Comment:
We should perhaps call this `maxDuration` or similar, to be more clear. It's a hard limit, not an exact duration.